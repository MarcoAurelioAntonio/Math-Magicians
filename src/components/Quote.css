.quote-container {
  width: 500px;
  height: auto;
  border: 1px solid grey;
}
