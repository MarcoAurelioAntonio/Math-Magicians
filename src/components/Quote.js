import React from 'react';
import './Quote.css';

const Quote = () => (
  <div className="quote-container">
    <p className="quote-text">
      Mathematics is not about numbers, equations, computations, or
      algorithms: it is about understanding.
      <span> -- William Paul Thurston</span>
    </p>
  </div>
);

export default Quote;
