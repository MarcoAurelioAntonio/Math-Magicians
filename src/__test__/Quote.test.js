import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Check tha Quote function renders content', () => {
  it('Check if render the Quote', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
