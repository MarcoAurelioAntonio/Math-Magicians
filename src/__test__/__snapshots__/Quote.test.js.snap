// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check tha Quote function renders content Check if render the Quote 1`] = `
<div
  className="quote-container"
>
  <p
    className="quote-text"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
    <span>
       -- William Paul Thurston
    </span>
  </p>
</div>
`;
